cmd/swarm/main.go

```diff
@@ -121,6 +121,10 @@ var (
 		Name:  "pss",
 		Usage: "Enable pss (message passing over swarm)",
 	}
+	DiscoveryEnabledFlag = cli.BoolFlag{
+		Name:  "discovery",
+		Usage: "Enable discovery",
+	}
```

Comment:
This flag should be true by default (consider naming it `NoDiscovery` like the [devp2p flag](https://github.com/ethersphere/go-ethereum/blob/network-testing-framework/cmd/utils/flags.go#L379-L382)).